### DISCUSSION TITLE
How do I allocate more RAM to the Minecraft server?

### DISCUSSION THREAD

**Discussion by CODEX-cpp** [AlexProgrammerDE/balena-minecraft-server#203] [Q&A]
I searched for the paper.jar file to change the RAM string. But in the whole Raspberry the file does not exist. then I have no idea how to give more ram to minecraft server

**Comment by viedok** [comment]
Hello,
Allocating more RAM to the server and a lot of other useful tweaks are set via variables directly in the balena dashboard.
You can see the whole list of variables [here](https://github.com/AlexProgrammerDE/balena-minecraft-server/blob/main/VARIABLES.md).

**Reply by CODEX-cpp** [reply]
thank you very much for your very helpful response
